components/actiongroup/stories/actiongroup.stories.js

```diff
@@ -1,11 +1,13 @@
 import { default as ActionButton } from "@spectrum-css/actionbutton/stories/actionbutton.stories.js";
+import { Sizes } from "@spectrum-css/preview/decorators";
 import { disableDefaultModes } from "@spectrum-css/preview/modes";
 import { size } from "@spectrum-css/preview/types";
 import pkgJson from "../package.json";
 import { ActionGroups } from "./actiongroup.test.js";
+import { OverflowOption, TreatmentTemplate } from "./template.js";
 
 /**
- * The action group component is a collection of action buttons.
+ * An action group is a grouping of [action buttons](?path=/docs/components-action-button--docs&globals=testingPreview:!true) that are related to each other.
  */
 export default {
 	title: "Action group",
```

Comment:
oooo! What do you think about adding an icon-only control? Similar to [tabs](https://github.com/adobe/spectrum-css/blob/7e8f6114aef049b82904cebf6a112aa5a2be0feb/components/tabs/stories/tabs.stories.js#L74)? That would give users the option to see their icon-only action buttons. 

<img width="589" alt="Screenshot 2024-10-02 at 2 12 43 PM" src="https://github.com/user-attachments/assets/9cf5ac44-62bf-457b-a027-b82f0e352505">